### ISSUE TITLE
FSDP SFT trainer with gradient accumulation, should loss not be divided by n_micro_batches before running loss.backward()

### ISSUE THREAD

**Issue by puneeshkhanna** [volcengine/verl#2919]
Check _compute_loss_and_backward implementation 
```
loss = torch.sum(loss) / (valid_token_this_rank + 1e-8) * dp_size   
if do_backward:
    loss.backward()
```

Before loss.backward(), we should add before -     
`loss = loss / n_micro_batches  # Average loss across micro-batches`

**Comment by Kirrito-k423** [comment]
定时清理检查结论 / Scheduled cleanup conclusion

该 issue 最近一次非当前运行账号的公开回复或引用是 2025-10-22T14:48:46Z（cross-referenced, actor: Fu-Fu-Fu-Fu），距 2026-07-15 已超过 6 个月。因此按定时清理标准关闭。若作者或社区成员仍能提供新的复现、证据或需求背景，可以继续评论请求重开，或创建新的 issue。

The latest public reply or reference from an account other than the current runner was 2025-10-22T14:48:46Z (cross-referenced, actor: Fu-Fu-Fu-Fu), which is more than 6 months before 2026-07-15. Therefore this is being closed under the scheduled cleanup rule. If the reporter or community can still provide a fresh reproduction, evidence, or requirement context, please comment to request reopening or open a new issue.

检查来源 / Checked sources:
- Issue: https://github.com/verl-project/verl/issues/2919
- Comments API: https://api.github.com/repos/verl-project/verl/issues/2919/comments
- Timeline API: https://api.github.com/repos/verl-project/verl/issues/2919/timeline
- Latest non-self public activity: https://github.com/verl-project/verl/issues/3864

This response was generated by a scheduled Codex GPT5.5 issue processing task.
